### PR TITLE
Add new CF for K8s area to CF on K8s WG

### PR DESCRIPTION
**Pull request by davewalter**
Add missing CF for K8s repositories to the CF on K8s working group.

### CODE REVIEW

**Comment by emalm** [comment]
If we're pulling in the cf-for-k8s repos for the time being until we retire them completely, should we have a new technical area for them that's separate from the existing "CF on K8s" one (which maybe we should also rename "Korifi" to match the new official project name)?

**Comment by davewalter** [comment]
@emalm We updated the PR to incorporate your suggestions.

On a separate topic, is there a process for removing approvers who are no longer full-time contributors to a project? We noticed that Akira and Mario are still listed as approvers for Korifi.

**Comment by emalm** [comment]
Thanks!

> is there a process for removing approvers who are no longer full-time contributors to a project?

Yep, https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#revoking-approver-role now represents the process that was approved in [RFC 0008](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0008-role-change-process.md).

**Review by Birdrock — APPROVED** [review]

File: toc/working-groups/cf-on-k8s.md [hunk 1]
```diff
@@ -68,12 +68,29 @@ areas:
   - name: Clint Yoshimura
     github: clintyoshimura
   repositories:
+  - cloudfoundry/cf-crd-explorations
```
Comment (emalm) [inline toc/working-groups/cf-on-k8s.md:71]: This repository is archived, so it shouldn't be under the ownership of any WG.
```suggestion
```

**Review by emalm — CHANGES_REQUESTED** [review]
Minor change requested to exclude the archived CRD repository, but otherwise LGTM!

**Review by emalm — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Pull request merged**